abb_bringup/launch/abb_moveit.launch.py

```diff
@@ -157,28 +157,24 @@ def generate_launch_description():
         DeclareLaunchArgument(
             "robot_xacro_file",
             description="Xacro describing the robot.",
-            choices=["irb1200_5_90.xacro"],
```

Comment:
Any reason not to just expand the choices to include what is needed? Is it not part of the current package?

Reply:
we have many `xacro` possibilities depending on which cell/robot/eef we're using and the same goes for `moveit_config` packages